src/components/Block/Results.vue

```diff
@@ -37,7 +37,9 @@ const hideAbstain = props.space?.voting?.hideAbstain ?? false;
     :title="ts >= proposal.end ? $t('results') : $t('currentResults')"
   >
     <div v-for="choice in choices" :key="choice.i">
-      <template v-if="!(proposal.type === 'basic' && hideAbstain && choice.i === 2)">
+      <template
```

Comment:
You shouldn't be changing linting so much. Please use the linting standards that we have set in the project.